ReactiveCocoa/Swift/Event.swift

```diff
@@ -63,6 +63,43 @@ public enum Event<T, E: ErrorType> {
 		}
 	}
 
+	/// Lifts the given function over the event's error.
+	public func mapError<F>(f: E -> F) -> Event<T, F> {
+		switch self {
+		case let .Next(value):
+			return .Next(value)
+
+		case let .Error(error):
+			return .Error(error.map(f))
+
+		case .Completed:
+			return .Completed
+
+		case .Interrupted:
+			return .Interrupted
+		}
+	}
+
+	/// Unwraps the contained `Next` value.
+	public var nextValue: T? {
```

Comment:
It does have a little potential to cause confusion, but in my mind I think of `.Next` as the only event that “has a value,” in the sense of something that I would do processing upon.

Not a very rigorous definition, I admit.